client/foundries.go

```diff
@@ -703,7 +703,8 @@ func (a *Api) TargetImageCreate(factory string, targetName string) (string, erro
 	if err := json.Unmarshal(*resp, &pr); err != nil {
 		return "", err
 	}
-	return pr.JobServUrl + "runs/assemble-system-image/console.log", nil
+	ciWebUrl := strings.Replace(pr.JobServUrl, "api", "ci", 1)
+	return ciWebUrl, nil
```

Comment:
I don't think changing api and ci produces the same URLs. And if the future the CI urls might be pointing into app.foundries.io for the stuff Milo is doing. I think we have to update the ota-lite API to return both kinds or URLs.